docs/quickstart.rst

```diff
@@ -86,10 +85,10 @@ At this point, nothing is running yet. We've just created a task, which is the t
 (or blueprint) for a job.
 
 Our task doesn't really have an impact on the world (a side effect). It doesn't write a
-file, or update a database, it doesn't make an API call. In the real life, this is a
+file, or update a database, it doesn't make an API call. In real life, this is a
 problem, because at this point, all the job is doing is wasting CPU cycle. In our case,
 though, we'll just monitor the standard output to see if our task executed successfully,
-we should be able to see the result of the "print" call
+and check the result of the "print" call
```

Comment:
This changes the meaning of the sentence :o The goal is to make sure a result is displayed. We are not checking the result in itself, but rather the presence of something in the standard output.

Reply:
I've changed it a bit :)